Sources/DataPipeline/DataPipelineImplementation.swift

```diff
@@ -39,6 +44,11 @@ class DataPipelineImplementation: DataPipelineInstance {
             analytics.add(plugin: customerIODestination)
         }
 
+        // plugin to add contextual information to device attributes
+        if let plugin = deviceUpdatePlugin {
+            analytics.add(plugin: plugin)
+        }
```

Comment:
Do we need to store it at class level? Or could this be simply follows? 🤔 

```suggestion
        if moduleConfig.autoTrackDeviceAttributes {
            analytics.add(plugin: DeviceContexualAttributes())
        }
```

Reply:
ahhh forgot we have module config here